incubator/binding-pgsql/src/test/java/io/aklivity/zilla/runtime/binding/pgsql/parser/PgsqlParserTest.java

```diff
@@ -52,7 +53,7 @@ public void shouldParseWithPrimaryKeySql()
     }
 
     @Test
-    public void shouldCreateParseWithPrimaryKeysSql()
+    public void shouldCreateTableParseWithPrimaryKeysSql()
```

Comment:
```suggestion
    public void shouldCreateTableWithPrimaryKey()
```
works?

Note: we are in the parser test class, so no need to use parse in the name of any of the test methods, please review all of the test method names.